elements/lisk-cryptography/src/buffer.ts

```diff
@@ -12,19 +12,38 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import * as BigNum from '@liskhq/bignum';
 
 export const BIG_ENDIAN = 'big';
 export const LITTLE_ENDIAN = 'little';
+const MAX_NUMBER_BYTE_LENGTH = 6;
 
 export const intToBuffer = (
 	value: number | string,
 	byteLength: number,
 	endianness: string = BIG_ENDIAN,
-) => new BigNum(value).toBuffer({ size: byteLength, endian: endianness });
+) => {
+	const buffer = Buffer.alloc(byteLength);
+	if (endianness === 'big') {
+		if (byteLength <= MAX_NUMBER_BYTE_LENGTH) {
+			buffer.writeUIntBE(Number(value), 0, byteLength);
+		} else {
+			buffer.writeBigUInt64BE(BigInt(value));
```

Comment:
Why not to provide offset in second condition?

Reply:
for the `writeUintBE` it is required to specify probably because byteLength is required, but on BigUint64BE, it's optional